### PR TITLE
payment icon for mbway

### PR DESCRIPTION
**Pull request by davidmfernandes**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by davidmfernandes** [comment]
Hi tauthomas01,
I've commited the svg file with the requested changes.

File: app/assets/images/payment_icons/mbway.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...335.128.393.01.035.017.071.017.109a.32.32 0 01-.099.243c-.067.064-.163.096-.289.096-.096 0-.164-.022-.213-.064a.404.404 0 01-.11-.177 11.55 11.55 0 01-.196-.553c-.073-.225-.153-.468-.235-.732H29.48" fill="#100f0d"/><path id="path26" d="M34.12 12.443a.654.654 0 01-.079-.091 5.456 5.456 0 01-.33-.514 9.736 9.736 0 01-.275-.497c-.09-.177-.177-.356-.262-.534a12.458 12.458 0 01-.231-.496c-.071-.155-.13-.289-.178-.4a1.671 1.671 0 01-.112-.298.428.428 0 01-.005-.062c0-.099.034-.182.103-.246a.395.395 0 01.286-.101c.11 0 .187.028.235.083.047.056.082.11.103.165.01.029.038.088.079.181l.152.346c.061.138.13.289.207.453.079.166.156.328.237.488.079.159.158.311.232.458.073.144.145.264.21.359a6.14 6.14 0 00.34-.603c.12-.242.236-.48.347-.716.11-.238.204-.447.286-.627.079-.182.127-.295.144-.339a.666.666 0 01.103-.165c.046-.055.126-.083.235-.083.122 0 .216.034.287.101a.327.327 0 01.101.246c0 .026 0 .048-.002.062-.003.014-.011.035-.02.062l-.096.239a24.658 24.658 0 01-.405.903c-.087.178-.174.355-.265.531-.091.175-.182.344-.274.503-.09.158-.175.295-.252.404a1.237 1.237 0 01-.156.188v1.707c0 .11-.035.199-.105.27a.37.37 0 01-.268.103.37.37 0 01-.268-.103.366.366 0 01-.104-.27v-1.707" fill="#100f0d"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/mbway.svg:1]: Can we remove all unused `id`? Like `id="path14"`, `id="path16", etc.

Comment (davidmfernandes) [reply app/assets/images/payment_icons/mbway.svg]: yes. I've committed the file with the changes.

**Comment by tauthomas01** [comment]
I'm seeing errors from travis PI. I'm not too sure what the error means, but in the meantime, could you rebase your branch off `master`?

**Comment by davidmfernandes** [comment]
@tauthomas01 could you please check whats wrong ?

**Comment by tauthomas01** [comment]
Hi @davidmfernandes, can you try and rebase your branch off master? It should work now.

**Comment by davidmfernandes** [comment]
@tauthomas01 is something else needed to change in order for the PR to be merged ?

**Review by tauthomas01 — APPROVED** [review]
Looks good.

**Pull request merged**